src/ase/cw/gui/ServerFrame.java

```diff
@@ -29,8 +34,8 @@ public ServerFrame(int id, JFrame parentFrame, OrderController controller) {
         this.setIconImage(Toolkit.getDefaultToolkit().getImage(getClass().getClassLoader().getResource("coffee.png")));
         this.setTitle("Server "+serverId);
         this.setName("QueueFrame "+this.getTitle());
-        this.setPreferredSize(new Dimension(300, 200));
-        this.setLocation(parentFrame.getX()+parentFrame.getWidth()+(serverId*10), parentFrame.getY()+(serverId*10));
+        this.setPreferredSize(new Dimension(dimWidth, dimHeight));
+        this.setLocation(parentFrame.getX()+parentFrame.getWidth()+gapX, parentFrame.getY()+(serverId-1)*(gapY+dimHeight));
```

Comment:
I am not a massive fan of this as it will become problematic to add new servers so may have to change this back if it causes problems